### PR TITLE
v1: add support for fedora identities

### PR DESCRIPTION
**Pull request by croissanne**

### CODE REVIEW

**Review by ezr-ondrej — APPROVED** [review]
Dunno how to test this, but looks sane and the fallback to RHID seems done well, so at minimum it should not break stage :)

I guess we could remove account IDs soon to simplify this :)

**Comment by croissanne** [comment]
> Dunno how to test this, but looks sane and the fallback to RHID seems done well, so at minimum it should not break stage :)
> 
> I guess we could remove account IDs soon to simplify this :)

Eh i tihnk it's fine to have as many fallbacks as we need. maybe at some point we have a little library which can handle multiple identities, idk.

File: internal/v1/server_identity.go [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+			// the user's org does not have an associated EBS account number, these
+			// are associated when a billing relationship exists, which is a decent
+			// proxy for RHEL entitlements
+			logrus.Error("RHEL entitlement not present in identity header")
+			return i.AccountNumber() != ""
+		}
+		return entitled.IsEntitled
+	}
+	return false
+}
```
Comment (lzap) [inline internal/v1/server_identity.go:83]: It looks like `v1` package is doomed to be the jack of all trades 😓

**Pull request merged**